src/style.css

```diff
@@ -0,0 +1,39 @@
+html {
+  margin: 0;
+}
+
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+}
+
+.stars {
+  display: flex;
+  flex-direction: row;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  margin-right: 4px;
+  background-position: center;
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  
+}
+
+.container .stars--1 .stars__star:first-child,
+.container .stars--2 .stars__star:nth-child(-n+2),
+.container .stars--3 .stars__star:nth-child(-n+3),
+.container .stars--4 .stars__star:nth-child(-n+4),
+.container .stars--5 .stars__star:nth-child(-n+5) {
+  background-image: url(./images/star-active.svg);
+}
+
+.container .stars .stars__star:nth-child(n6) {
+  margin-right: 0;
+}
```

Comment:
1. Are you sure you need this style here.
2. Don't use selectors with increased specificity.